src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +33,11 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+	/**
+	 * @return the feedbackToUser
```

Comment:
If the method is so simple, it's better to write method description then use `@return`, e.g. `Returns ...`. Refer to #188 .